funtofem/interface/fun3d_interface.py

```diff
@@ -65,7 +65,6 @@ def __init__(
             MPI communicator
         model: :class:`FUNtoFEMmodel`
             FUNtoFEM model. This instantiatio
-            TODO
```

Comment:
Can we keep the TODO there or add the documentation to the parameters section for the remaining parameters?